release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3692,4 +3692,23 @@ $ oc adm release info 4.10.45 --pullspecs
 [id="ocp-4-10-45-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-46"]
+=== RHBA-2022:9099 - {product-title} 4.10.46 bug fix and security update
```

Comment:
Are we still using 2022 Errata even though it's 2023? I'm assuming yes and that it wasn't been changed on that side of things yet!

Reply:
Yeah that is what they say 🤷‍♀️ I was wondering that too earlier

Reply:
@opayne1 pending preview, this LGTM